xrootd/xrootd_config.go

```diff
@@ -99,6 +99,20 @@ func CheckXrootdEnv(origin bool) error {
 			" to desired daemon user %v", runtimeDir, username)
 	}
 
+	// The scitokens library will write the token cache into the user's home
+	// directory.  By setting $XDG_CACHE_HOME, we will move the cache into our runtime dir.
```

Comment:
Took me a few seconds to realize which "cache" this was referring to, due to overloaded terms. I was thinking the pelican cache for a second rather than the token cache. Maybe rename things to be a bit more specific? "tokenCacheDir" or something like that? Mostly for someone taking a look at this code a year from now.